front/cypress/e2e/light_authentication_flow.cy.ts

```diff
@@ -0,0 +1,80 @@
+import { randomString, randomEmail } from '../support/commands';
+
+describe('Light authentication flow', () => {
+  let projectId = '';
+  const projectTitle = randomString();
+  const projectDescriptionPreview = randomString();
+  const projectDescription = randomString();
+
+  before(() => {
+    cy.apiCreateProject({
+      type: 'continuous',
+      title: projectTitle,
+      descriptionPreview: projectDescriptionPreview,
+      description: projectDescription,
+      publicationStatus: 'published',
+      participationMethod: 'ideation',
+    }).then((project) => {
+      projectId = project.body.data.id;
+
+      cy.intercept(`**/projects/${projectId}/permissions/posting_idea`).as(
+        'setPermissionRequest'
+      );
+
+      cy.setAdminLoginCookie();
+      cy.visit(`/admin/projects/${projectId}/permissions`);
+      cy.get('#e2e-permission-email-confirmed-users').click();
+      cy.wait('@setPermissionRequest').then(() => {
+        cy.logout();
+      });
+    });
+  });
+
+  it('works when signing up with new email', () => {
+    cy.visit(`/projects/${projectTitle}`);
+    cy.wait(2000).then(() => {
```

Comment:
Could we remove the "wait" here and on line 60 and check until the e2e-idea-button exists instead (it will re-check many times until it loads + it finds it)? I've noticed the wait can be flaky on CI sometimes since CI tends to load things slowly.

```
cy.get('#e2e-idea-button').should('exist');
cy.get('#e2e-idea-button').click();

```

Reply:
The problem doesn't seem to be that the button doesn't exist yet- it's that somehow, if you click it too fast, it doesn't work. So for now I also added the `.should('exist')` bit, but also kept the `wait`

Reply:
Ahh ok, and `cy.get('#e2e-idea-button').click({force: true);` doesn't work either? :thinking: